source/sass/components/article-page.scss

```diff
@@ -0,0 +1,316 @@
+.article-summary {
```

Comment:
There is potentially alot of technical debt in this css. Some of the values being pulled, like font-family, are already defined as variables. 

The same may be true for some of these colors, and for the ones where they aren't, ideally we'd check with design and verify that they are adding new colors to the palette for the site, make new variables, and add them to the style guide.

As much as possible, we should use the Mozilla Bootstrap theme to keep things consistent. It's totally possible that there are aspects of the design spec that don't match what is defined for the site at large; for better long term maintenance it would be best for the Mozilla team to be able to reconcile all these discrepancies. 

All that being said, given our time frame it may be better to not count these issues as blocking.

Reply:
I'd defer to @mmmavis on situations like this. If we need to get this merged to unblock other PRs, let's make sure we open a follow-up issue to discuss potential improvements and/or work to stick to the styleguide